gui/default/syncthing/core/syncthingController.js

```diff
@@ -2530,6 +2530,11 @@ angular.module('syncthing.core')
                                 table: {
                                     indentation: 24,
                                 },
+                                strings: {
+                                  loading: $translate.instant("Loading..."),
```

Comment:
Please use consistent indentation of four spaces per level.

Reply:
Fixed. That was my sloppiness caused by a bad copy & paste and relying on the code editor to fix the indentation (which it did wrongly).